### PR TITLE
fix to map an undefined tensor back to a tensor list

### PR DESCRIPTION
**Pull request by davidberard98**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #75262

Taken from https://github.com/pytorch/pytorch/pull/60516

Differential Revision: [D35410201](https://our.internmc.facebook.com/intern/diff/D35410201)

### CODE REVIEW

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35410201).

**Comment by davidberard98** [comment]
@davidberard98 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35410201).

File: torch/csrc/jit/passes/lower_grad_of.cpp [hunk 1]
```diff
@@ -25,6 +25,12 @@ void LowerGradOf(Graph& g) {
                        ->insertBefore(else_block->return_node())
                        ->output();
       for (size_t i = 0; i < it->outputs().size(); ++i) {
+        // the else block returns a tensor for each of the outputs of the GradOf
```
Comment (Krovatkin) [inline torch/csrc/jit/passes/lower_grad_of.cpp:28]: love the comment!

**Review by Krovatkin — APPROVED** [review]
:shipit:

**Pull request closed without merging**